DatePicker/DatePicker.module.less

```diff
@@ -1,9 +1,29 @@
 // DatePicker.module.less
 //
+
 .datePicker {
 	.day,
 	.month,
 	.year {
 		/* These should be available to the JS but have no associated rules at this time. */
+		position: relative;
+
+		&:not(:last-child)::after {
+			content: '';
+			width: 3px;
+			height: 21px;
+			opacity: 0.4;
+			background-color: white;
+			display: inline-block;
+			vertical-align: inherit;
+			position: absolute;
+			top: 50%;
+			transform: translateY(-50%);
+			left: calc(100% - 3px);
+		}
```

Comment:
These are really specific. Could this be generalized at all? It's got colors, measurements, everything.

Also, it's targeting everything except the last element? I assume to put lines between each of the elements. I usually like to avoid using `:not()` since it works so differently from the way people expect. This could be refactored to use `nth-child` instead or the `+` operator to target everything after the first element; positioning the line to the left, instead of the right of each of the elements.

It's also both positioned with absolute positioning and set to inline-block? But then it also has a `vertical-align`?

Reply:
removed  display and vertical-align. 
When it comes to repositioning the common styling, I moved ths style to DateTime.module.less. I hope it's the good way of doing :)